src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -493,7 +493,7 @@ public void InvalidPublicKey()
             var r = MetadataReference.CreateFromStream(new MemoryStream(TestResources.SymbolsTests.Metadata.InvalidPublicKey, writable: false));
             Assert.Equal(CodeAnalysisResources.InMemoryAssembly, r.Display);
 
-            Assert.Throws<BadImageFormatException>(((AssemblyMetadata)r.GetMetadata()).GetAssembly);
+            Assert.Throws<BadImageFormatException>(() => ((AssemblyMetadata)r.GetMetadata()).GetAssembly());
```

Comment:
@KevinH-MS, The method call had changed to Assert.Throws<T>(Func<Task>). I figured it would be easier to make it into a Action call rather than modifying it to a Task call and awaiting.